actor/src/main/scala/org/apache/pekko/dispatch/VirtualThreadSupport.scala

```diff
@@ -0,0 +1,75 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.pekko.dispatch
+
+import org.apache.pekko.annotation.InternalApi
+import org.apache.pekko.util.JavaVersion
+
+import java.lang.invoke.{ MethodHandles, MethodType }
+import java.util.concurrent.{ ExecutorService, ThreadFactory }
+import scala.util.control.NonFatal
+
+@InternalApi
+private[dispatch] object VirtualThreadSupport {
+  private val lookup = MethodHandles.lookup
```

Comment:
can you use [public lookup](https://docs.oracle.com/javase/8/docs/api/java/lang/invoke/MethodHandles.html#publicLookup--)?